config/locales/en.yml

```diff
@@ -30,9 +30,16 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
+
   hello: "Hello world"
+  methodologies: "Methodologies"
+  recording: "Recording"
+  focus: "Focus"
+  researcher: "Researcher"
+  incentive: "Incentive"
+  age: "Age"
```

Comment:
We probably don't need translations for these; [`Symbol#humanize`](http://api.rubyonrails.org/classes/ActiveSupport/Inflector.html#method-i-humanize) would do the same.